src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx

```diff
@@ -48,13 +54,18 @@ export function ClimateSettingScheduleForm({
 function Content({
   onBack,
 }: Omit<ClimateSettingScheduleFormProps, 'className'>): JSX.Element {
-  const { control, watch } = useForm({
+  const { control, watch, resetField } = useForm({
     defaultValues: {
       deviceId: '',
       name: '',
       startDate: '',
       endDate: '',
       timeZone: getSystemTimeZone(),
+      hvacModeSetting: 'heat_cool' as HvacModeSetting,
```

Comment:
Interesting, what happens if this is not type cast?
```suggestion
      hvacModeSetting: 'heat_cool',
```